src/neo/Network/P2P/Capabilities/FullNodeCapability.cs

```diff
@@ -5,28 +5,34 @@ namespace Neo.Network.P2P.Capabilities
     public class FullNodeCapability : NodeCapability
     {
         public uint StartHeight;
+        public int MemPoolCount;
```

Comment:
I will remove this because it send 4 bytes more, and if you have 0 tx, you will spend less in a new message